cli/eva_sub_cli.py

```diff
@@ -1,20 +1,70 @@
+import os
 from argparse import ArgumentParser
 
+from ebi_eva_common_pyutils.config import WritableConfig
 from ebi_eva_common_pyutils.logger import logging_config
 
+from cli import SUB_CLI_CONFIG_FILE, __version__
+from cli.docker_validator import DockerValidator, docker_path, container_image
 from cli.submit import StudySubmitter
 
+VALIDATION_OUTPUT_DIR = "validation_output"
+VALIDATE = 'validate'
+SUBMIT = 'submit'
+RESUME = 'resume'
+
+logging_config.add_stdout_handler()
```

Comment:
The call to `add_stdout_handler ` should be made in the `main` function. This avoids having multiple handlers being added if this file is ever loaded as a library.

Reply:
updated